### PR TITLE
fix: clear localStorage and sessionStorage

### LINKED ISSUE #5326: Replace vuex-settings to static in code
Seems like we have a ton of the issues that prefixes are heavily cached on the client a lot of people experience issues with cache.

photo_2023-03-22 19 06 48

Tasks:
- [ ] move `static` package from kodadot/packages here as a new workspace folder
- [ ] where possible (except Settings store - will be separated issue) 

ht/ @DeluneClaire, @yangwao

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #5326

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

https://user-images.githubusercontent.com/734428/227542183-8b46f52f-d0bb-4453-9826-5dd6ab464699.mov

### CODE REVIEW

**Review by roiLeo — APPROVED** [review]
hmm.. if it helps why not.
maybe pwa setup? I haven't seen the "New version" notification for a while.

File: utils/cachingStrategy.ts [hunk 1]
```diff
@@ -62,12 +62,16 @@ export const flushIndexedDb = () => {
...
+      window.localStorage.clear()
+      flushIndexedDb()
+    }, 100)
   } catch (error) {
     console.error(error)
   } finally {
-    window.location.reload()
+    setTimeout(() => {
+      window.location.reload()
+    }, 200)
```
Comment (Jarsen136) [inline utils/cachingStrategy.ts:75]:
```
export const clearSession = () => {
  try {
      window.sessionStorage.clear()
      window.localStorage.clear()
      flushIndexedDb()
  } catch (error) {
    console.error(error)
  } finally {
    setTimeout(() => {
      window.location.reload()
    }, 100)
}
```
How about this one? Will it do the same thing

**Review by Jarsen136 — COMMENTED** [review]
Looks like an easier solution to solve the cache issue.

File: utils/cachingStrategy.ts (hunk 1 above)
Comment (Jarsen136) [reply utils/cachingStrategy.ts:75]: Aright. Let it go

Comment (roiLeo) [reply utils/cachingStrategy.ts:75]: frozen_wide-a051d4ad0bc2b62f4976fc58d9a963450e63b359

**Comment by vikiival** [comment]
pay 15 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $15 @ 33.83 USD/KSM ~ 0.443 $KSM
🧗 [DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM](https://kusama.subscan.io/account/DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)
🔗 [0x15d1a7c482df78fd3b5bcf9df7766f42cad7c7fbd7e79eee28994a3b3a6e91c6](https://kusama.subscan.io/extrinsic/0x15d1a7c482df78fd3b5bcf9df7766f42cad7c7fbd7e79eee28994a3b3a6e91c6)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**